pages/account.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import styles from "../styles/pages/account.module.scss";
+import AccountUserName from "../components/account/AccountUserName";
+
+export default function Account() {
+	const daysAlive = 9;
+	const [profilePicSrc, setProfilePicSrc] = useState("/lio_img.png");
+
+	const onUpload = (event: React.ChangeEvent<HTMLInputElement>) => {
+		try{
+			if(event.target.files){
+			const file = event.target.files[0];
+			setProfilePicSrc(URL.createObjectURL(file));
+		}else{
```

Comment:
not gonna sugarcoat it my friend, the formatting here is horrendous - the else is on the same indent as the try/catch block and the content of the if is not indented...
Shift+alt+F kitser.

Reply:
noted